src/pgexporter_ext/lib.c

```diff
@@ -485,17 +485,21 @@ os_info(Tuplestorestate* tupstore, TupleDesc tupdesc)
       nulls[OS_INFO_DOMAIN_NAME] = true;
    }
 
-   os_info_file = fopen("/etc/system-release", "r");
+   os_info_file = fopen("/etc/os-release", "r");
 
    if (!os_info_file)
    {
       nulls[OS_INFO_NAME] = true;
    }
    else
    {
-      if (fgets(&buffer[0], max_length, os_info_file) != NULL)
+      snprintf(&os_name[0], sizeof(os_name), "%s", "Linux");
+      while (fgets(&buffer[0], max_length, os_info_file) != NULL)
       {
-         memcpy(os_name, buffer, strlen(buffer) - 1);
+         if(strstr(buffer, "PRETTY_NAME") != NULL){
```

Comment:
Hmm, it should be

```
if (strstr() != NULL)
{
```

Reply:
@jesperpedersen You're referring to a code formatting error here right?

Reply:
Yeah